timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResultTimeline.vue

```diff
@@ -485,12 +405,35 @@ export default {
     },
     checkAnalyzerOutput: function () {
       try {
-        JSON.parse(this.timeline.results[0].verdict)
+        JSON.parse(this.timelineFirstResult.verdict)
         return true
       } catch (e) {
         return false
       }
     },
+    timelineFirstResult: function () {
+      return this.timeline && this.timeline.results.length > 0 && this.timeline.results[0]
+        ? this.timeline.results[0]
+        : '... no results found'
```

Comment:
If you return a string here, it will not work as expected.

in `timelineCreated` you check `if (!firstEntry)`, which will never be the case. To fix this, you could return undefined here. 

In any case all the places where you call `timelineFirstResult.verdict` will cause errors. So it might be a good idea to either check in those places via `this.timelineFirstResult && this.timelineFirstResult.verdict` or to create a computed property for this.